server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -93,6 +98,32 @@ private void extensionsDiscovery() throws IOException {
         for (final Path plugin : PluginsService.findPluginDirs(extensionsPath)) {
             try {
                 PluginInfo pluginInfo = PluginInfo.readFromProperties(plugin);
+
+                // TODO: Add unit tests for commented section below
```

Comment:
Do we have issue for unit tests?

Reply:
That's my next task.  I'll create one once I merge this PR